src/staticanalysis/frontend/src/Tasks.vue

```diff
@@ -114,7 +114,7 @@ export default {
             <p v-if="task.data.title">{{ task.data.title }}</p>
             <p class="has-text-danger" v-else>No title</p>
             <p>
-              <small class="mono has-text-grey-light">{{ task.data.diff_phid}}</small>
+              <small class="mono has-text-grey-light">{{ task.data.diff_phid}}</small> - rev {{ task.data.diff_id}}
```

Comment:
Please add a default value to display, as the current tasks displayed won't have the `diff_id`:

```javascript
{{ task.data.diff_id || 'unknown' }}
```

Reply:
And replace the `rev` by `diff` here, makes more sense